src/pages/export-page/ExportPage.tsx

```diff
@@ -104,80 +104,82 @@ export const ExportPage = () => {
 
   return (
     <>
-      <h1 className="govuk-heading-l govuk-!-margin-top-6">
-        Export your timetable
-      </h1>
-
-      <p className="govuk-body">
-        When you have completed all the details, you’ll need to export your data
-        as CSV files. You can use these files to generate the timetable on your
-        website.
-      </p>
-
-      <p className="govuk-body">
-        <strong>Step 1</strong>
-      </p>
-
-      <p className="govuk-body">
-        You need to export <strong>both</strong> CSV files.
-      </p>
-
-      <p className="govuk-body">
-        <strong>Step 2</strong>
-      </p>
-
-      <p className="govuk-body">
-        Next, you’ll need to follow the guidance which explains how to use the
-        CSV files you’ve just downloaded to publish the timetable on your
-        website:{" "}
-        <Link className="govuk-link" to={PageRoute.PublishOnlineGuidance}>
-          Publishing a local plan timetable online: guidance for web teams
-        </Link>
-      </p>
-
-      <div>
-        <a
-          role="button"
-          type="button"
-          data-testid="csv-download-button"
-          href={timetableEventsDownloadLink}
-          download="timetable.csv"
-        >
-          <Button>Export timetable CSV</Button>
-        </a>
+      <div className="govuk-body govuk-!-width-two-thirds">
```

Comment:
on this page only the top part is 2/3 width. at the bottom the planViewer spans the whole width of the page